frontend/src/components/Scopes.tsx

```diff
@@ -87,10 +87,17 @@ const Scopes: React.FC<ScopesProps> = ({
   }, [capabilities.scopes]);
 
   const [scopeGroup, setScopeGroup] = React.useState<ScopeGroup>(() => {
-    if (isolated && scopes.length === capabilities.scopes.length) {
+    if (
+      isolated &&
+      scopes.length === fullAccessScopes.length &&
+      scopes.every((scope) => fullAccessScopes.includes(scope))
```

Comment:
unrelated to this PR but shouldn't this be `isolatedScopes`? Since it doesn't have `sign_message` (is that intentional btw)

Reply:
fixed, thanks!

Reply:
it's intentional to not give a sub wallet sign_message currently (although we do say sub-wallets should still be used by "trusted" users)